src/.eslintrc.json

```diff
@@ -1,4 +1,8 @@
 {
+	"env": {
+		"browser": false,
```

Comment:
Ah, I get it now. I'll update the others as well. I'd only leave the ones that explicitly extend `src/.eslintrc.json` as there we do want to replicate the same settings for source files.